### PR TITLE
added unit tests for websocket plugin.

### PR DESCRIPTION
**Pull request by tsawan**

### CODE REVIEW

**Comment by tsawan** [comment]
@amcdnl I ran test:ci and test:ci:integration locally before submitting this PR.
Please check the error message. 
error An unexpected error occurred: "Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.".

I am not sure I can do anything about it.

File: packages/websocket-plugin/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "version": "0.0.0",
   "peerDependencies": {
     "@ngxs/store": "^0.0.0",
-    "@angular/core": "^5.0.0"
+    "@angular/core": "^5.0.0",
+    "mock-socket": "^7.1.0"
   }
```
[the remark this answers is not in the document]
Comment (amcdnl) [reply packages/websocket-plugin/package.json]: Agreed.

[the remark this answers is not in the document]
Comment (tsawan) [reply packages/websocket-plugin/package.json]: I removed it and now it's giving "Cannot find module 'mock-socket'." during CI build

**Comment by amcdnl** [comment]
I took your branch and fixed it. PR'd and merged.

**Pull request closed without merging**